lib/util/utils.ts

```diff
@@ -110,28 +127,15 @@ export async function parseJson(
         "https://raw.githubusercontent.com$2$3"
       )
     }
-    const res = makeRequest({ url: specPath, errorOnNon200Response: true })
+    const res = makeRequest({ url: specPath, errorOnNon200Response: true }).then(applySuppression)
```

Comment:
why not async/await?

Reply:
This is clearer imo

```
const swaggerObject = await makeRequest({ url: specPath, errorOnNon200Response: true })
docCache[specPath]  = applySuppression(res)
return docCache[specPath] 
```

Reply:
or maybe i mean

```
const swaggerObject = await makeRequest({ url: specPath, errorOnNon200Response: true })
applySuppression(swaggerObject)
docCache[specPath]  = Promise.resolve(swaggerObject)
return swaggerObject
```

Reply:
We shouldn't await (block) for uploading document from HTTPS.

Reply:
we do this already though... see line 132